apps/web-giddh/src/app/ledger/components/update-ledger-entry-panel/update-ledger-entry-panel.component.ts

```
@@ -3096,4 +3096,14 @@ export class UpdateLedgerEntryPanelComponent implements OnInit, AfterViewInit, O
         });
     }
+
+    /**
+     * This will be use for duplicate entry
+     *
+     * @memberof UpdateLedgerEntryPanelComponent
+     */
+    public duplicateEntry(): void {
+        this.isDuplicateEntry = true;
+        this.closeUpdateLedgerModal.emit();
+    }
```

Biome (2.1.2)

[error] Duplicate class member name "duplicateEntry"



(lint/suspicious/noDuplicateClassMembers)